es-app/src/Gamelist.cpp

```diff
@@ -165,7 +165,7 @@ void addFileDataNode(pugi::xml_node& parent, const FileData* file, const char* t
 		//there's something useful in there so we'll keep the node, add the path
 
 		// try and make the path relative if we can so things still work if we change the rom folder location in the future
-		newNode.prepend_child("path").text().set(Utils::FileSystem::createRelativePath(file->getPath(), system->getStartPath(), false).c_str());
+		newNode.prepend_child("path").text().set(Utils::FileSystem::createRelativePath(file->getPath(), system->getStartPath(), false, false).c_str());
```

Comment:
Why change this to "false", though? It has nothing to do with "Trusted gamelists" and this code runs on save. Can you elaborate?

Reply:
Because `system->getStartPath()` is always a folder there is no reason to hit the file system in the read or write case.

If there's some concern about it not being a folder, and I don't know what that would be, checking it once up front would be far better than checking it every single file of the game list.